x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClustersQueryIT.java

```diff
@@ -728,6 +729,17 @@ public void testWarnings() throws Exception {
         assertTrue(latch.await(30, TimeUnit.SECONDS));
     }
 
+    public void testSearchesWhereClusterExpressionCannotBeResolved() {
+        setupTwoClusters();
+
```

Comment:
Let's also add a test that tests:

1) multiple invalid index expressions, such as "FROM nosuchclusterA*, nosuchclusterB* | xxx" (VerificationException expected)
2) where it is paired with a valid local index: "FROM nosuchcluster*,logs*"  -- there might already be test that does this in this test suite - can you confirm if that's right? If not, add one here.